ooniapi/services/ooniprobe/src/ooniprobe/routers/reports.py

```diff
@@ -187,10 +141,11 @@ async def receive_measurement(
     data = await request.body()
     if content_encoding == "zstd":
         try:
+            compressed_len = len(data)
             data = zstd.decompress(data)
-            ratio = len(data) / len(data)
+            ratio = compressed_len / len(data)
             log.debug(f"Zstd compression ratio {ratio}")
-        except Exception as e:
+        except Exception:
             log.info("Failed zstd decompression")
```

Comment:
Can we log `e` inside of `log.info`. Make sure you use `{e}` inside of `log.info(f"")` which should perform sanitisation.